doc/admin/install/google_cloud.md

```diff
@@ -24,23 +24,48 @@ If you're just starting out, we recommend [installing code-server locally](../..
 ```
 gcloud compute ssh --zone [region] [instance name]
 ```
-2. Download the binary using the link we copied to clipboard
+
+2. Find the latest Linux release from this URL:
+```
+https://github.com/codercom/code-server/releases/latest
+```
+
+3. Replace {version} in the following command with the version found on the releases page and run it (or just copy the download URL from the releases page):
+```
+wget https://github.com/codercom/code-server/releases/download/{version}/code-server-{version}-linux-x64.tar.gz
 ```
-wget https://github.com/codercom/code-server/releases/download/0.1.4/code-server-linux
+
+4. Extract the downloaded tar.gz file with this command, for example:
 ```
-3. Make the binary executable if you run into any errors regarding permission:
-  ```
-  chmod +x code-server-linux
-  ```
-> To ensure the connection between you and your server is encrypted view our guide on [securing your setup](../../security/ssl.md)
-4. Start the code-server
+tar -xvzf code-server-{version}-linux-x64.tar.gz
+```
+
+5. Navigate to extracted directory with this command:
+```
+cd code-server-{version}-linux-x64
+```
+
+6. Make the binary executable if you run into any errors regarding permission:
+```
+chmod +x code-server
+```
+
+> To ensure the connection between you and your server is encrypted view our guide on [securing your setup](../security/ssl.md)
+
+7. Start the code-server
 ```
 sudo ./code-server-linux -p 80
 ```
+
 > For instructions on how to keep the server running after you end your SSH session please checkout [how to use systemd](https://www.linode.com/docs/quick-answers/linux/start-service-at-boot/) to start linux based services if they are killed
-5. Access code-server from the public IP of your Google Cloud instance we noted earlier in your browser. 
+
+8. Access code-server from the public IP of your Google Cloud instance we noted earlier in your browser. 
```

Comment:
Use markdown number list (generate auto sequence)

```
1. one
1. two
1. ...
```

Render:
1. one
1. two
1. ...

Reply:
I've changed that to an unordered list now to be same as other docs.